protoc-gen-grpc-gateway/gengateway/generator_test.go

```diff
@@ -9,7 +10,16 @@ import (
 	"github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway/descriptor"
 )
 
-func TestGenerateServiceWithoutBindings(t *testing.T) {
+func newExampleFileDescriptor() *descriptor.File {
+	return newExampleFileDescriptorWithGoPkg(
+		&descriptor.GoPackage{
+			Path: "example.com/path/to/example/example.pb",
```

Comment:
Shouldn't the file name end in .go?

Reply:
I don't think so.

I migrated an already existing example into a customisable function without doing any change to it but as you can observe in the two cases that I've implemented i used this descriptor as a 1-1 representation of the `go_package` proto option.

I tried to touch as less as possible :)